src/test/index.ts

```diff
@@ -87,6 +91,8 @@ export const test = _baseTest.extend<PlaywrightTestArgs & PlaywrightTestOptions,
       options.bypassCSP = bypassCSP;
     if (colorScheme !== undefined)
       options.colorScheme = colorScheme;
+    if (reducedMotion !== undefined)
```

Comment:
What does this all code do? :)

Reply:
Pass it from the worker options to the context options.

Reply:
is this a drive-by?

Reply:
It is 👍